### PR TITLE
fix the error after updating country, still set as Australia

### PR DESCRIPTION
**Pull request by Zhaolian-Zhou**

### CODE REVIEW

File: protected/views/patient/_form_address.php [hunk 1]
```diff
@@ -60,8 +60,11 @@
 <div class="row field-row">
     <div class="large-3 column"><?php echo $form->labelEx($address,'country_id'); ?></div>
     <div class="large-4 column end">
-
-        <?php echo $form->dropDownList($address,'country_id', $countries, array('options'=>array(array_search('Australia', $countries)=>array('selected'=>true)))); ?>
+        <?php if (isset($address["country_id"])){
+            echo $form->dropDownList($address,'country_id', $countries, array('options'=>array($address["country_id"]=>array('selected'=>true))));
+        }else{
```
Comment (apigram) [inline protected/views/patient/_form_address.php]: Whitespace needs to be fixed here. Should be a space separating braces from keywords and conditions

**Review by apigram — COMMENTED** [review]
CTRL+ALT+L should fix the formatting of the braces.

File: protected/views/patient/_form_address.php [hunk 2]
```diff
@@ -60,9 +60,9 @@
 <div class="row field-row">
     <div class="large-3 column"><?php echo $form->labelEx($address,'country_id'); ?></div>
     <div class="large-4 column end">
-        <?php if (isset($address["country_id"])){
+        <?php if (isset($address["country_id"])) {
             echo $form->dropDownList($address,'country_id', $countries, array('options'=>array($address["country_id"]=>array('selected'=>true))));
-        }else{
+        }else {
```
Comment (apigram) [inline protected/views/patient/_form_address.php]: There needs to be a space before the else keyword as well. Not sure why CTRL+ALT+L didn't fix that,

Comment (Zhaolian-Zhou) [reply protected/views/patient/_form_address.php]: Seems the code styling doesn't work in my PHPStorm, I am trying fix it.
Will change this manually first.

**Review by apigram — APPROVED** [review]

**Comment by marshl** [comment]
@Zhaolian-Zhou You can delete the fix branch when you are done with it

**Pull request merged**